examples/lighting-app/silabs/src/LightingManager.cpp

```diff
@@ -307,3 +359,56 @@ void LightingManager::OnTriggerOffWithEffect(OnOffEffect * effect)
     LightMgr().mOffEffectArmed = true;
     LightMgr().StartTimer(offEffectDuration);
 }
+
+#if (defined(SL_MATTER_RGB_LED_ENABLED) && SL_MATTER_RGB_LED_ENABLED == 1)
+bool LightingManager::InitiateLightctrlAction(int32_t aActor, Action_t aAction, uint32_t aAttributeId, uint8_t * value)
```

Comment:
Nit: PascalCasing the c of Ctrl should be uppercase 
```suggestion
bool LightingManager::InitiateLightCtrlAction(int32_t aActor, Action_t aAction, uint32_t aAttributeId, uint8_t * value)
```